src/Mobs/Spider.cpp

```diff
@@ -43,9 +43,15 @@ void cSpider::EventSeePlayer(cEntity * a_Entity)
 		return;
```

Comment:
Does it hurt performance the way it is? Lighting for the sake of spiders is not urgent. If I bailout without calling `GetWorld()->QueueLightChunk(GetChunkX(), GetChunkZ());`, would it be eventually be lighted?